content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -122,7 +122,7 @@ secrets:
 
 then you will see that a token has automatically been created and is referenced by the service account.
 
-You may use authorization plugins to [set permissions on service accounts](/docs/admin/authorization/#a-quick-note-on-service-accounts).
+You may use authorization plugins to [set permissions on service accounts](docs/reference/access-authn-authz/authorization/).
```

Comment:
I'd suggest we use /docs/reference/access-authn-authz/rbac/#service-account-permissions